components/Currency/CurrencyIcon/CurrencyIcon.tsx

```diff
@@ -91,7 +91,7 @@ export const CurrencyIcon: FC<CurrencyIconProps> = ({ currencyKey, type, ...rest
 						src={
 							synthetixTokenListMap != null && synthetixTokenListMap[currencyKey] != null
 								? synthetixTokenListMap[currencyKey].logoURI
-								: getSynthIcon(currencyKey)
+								: getSynthIcon(currencyKey as CurrencyKey)
```

Comment:
this is the kind of issue you get unless you remove `CurrencyKey` everywhere in the dapp

Reply:
If you really want to remove it that is fine but then you should remove all instances so you don't need the as keyword here